x-pack/plugins/fleet/common/openapi/entrypoint.yaml

```diff
@@ -16,24 +16,37 @@ paths:
   # plugin-wide endpoint(s)
   /setup:
     $ref: paths/setup.yaml
+  /setup/preconfiguration:
+    $ref: paths/setup_preconfiguration.yaml
   /settings:
     $ref: paths/settings.yaml
+  # App endpoints
+  /service-tokens:
+    $ref: paths/service_tokens_deprecated.yaml
+  /service_tokens:
+    $ref: paths/service_tokens.yaml
   # EPM / integrations endpoints
+  /epm/_bulk:
+    $ref: paths/epm@bulk.yaml
   /epm/categories:
     $ref: paths/epm@categories.yaml
+  /epm/packages/limited:
+    $ref: paths/epm@limited_list.yaml
   /epm/packages:
     $ref: paths/epm@packages.yaml
   '/epm/packages/{pkgkey}':
-    $ref: 'paths/epm@packages@{pkgkey}.yaml'
-    deprecated: true
+    $ref: 'paths/epm@packages@{pkgkey}_deprecated.yaml'
   '/epm/packages/{pkgName}/{pkgVersion}':
-    $ref: 'paths/epm@packages@{pkg_name}@{pkg_version}.yaml'  
+    $ref: 'paths/epm@packages@{pkg_name}@{pkg_version}.yaml'
```

Comment:
pkgVersion param in this file should be made optional

Reply:
I am not sure how to mark it as optional. I tried changing the` required: true` to `false` but it gives an error, then I saw in the swagger docs that the parameters in a path should always be marked as required:
https://swagger.io/docs/specification/describing-parameters/#path-parameters

Reply:
hm interesting, so we might need to keep a separate path `/epm/packages/{pkgName}`
this is currently included in deprecated path with {pkgKey}
should we raise a new issue, so that when we remove deprecated code, we remember to keep the path above?

Reply:
Yes, I think that we need to create a new file and mark the old as deprecated. I agree with having an ticket to remember it in the future.

Reply:
Raised this: https://github.com/elastic/kibana/issues/121485